### DISCUSSION TITLE
Bad credentials for user

### DISCUSSION THREAD

**Discussion by bokket** [pgagroal/pgagroal#352] [General]
I have gone through the [01_install ](https://github.com/agroal/pgagroal/blob/master/doc/tutorial/01_install.md)tutorial.

When I use `pgagroal-cli status` to check connection details.It tells me bad credentials for user.What I missed?
```shell
pgagroal-cli -h localhost -p 2345 -U myuser details

Password : 
pgagroal-cli: Bad credentials for myuser

```

But I use the "application password" to access the PostgreSQL instance and make it.The Log is not error.
```shell
psql -h localhost -p 2345 -U myuser mydb  
Password for user myuser: 
psql (12.14 (Ubuntu 12.14-0ubuntu0.20.04.1))
Type "help" for help.

mydb=> 
```

```shell
2023-03-20 14:08:40 INFO  main.c:1085 pgagroal: 1.6.0 started on *:2345
```

It's my configure file:
```shell
[pgagroal]
host = *
port = 2345

log_type = file
log_level = info
log_path = /tmp/pgagroal.log

max_connections = 100
idle_timeout = 600
validation = off
unix_socket_dir = /tmp/

management = 2347

[primary]
host = localhost
port = 5432
```

**Comment by jesperpedersen** [comment]
The `:2345` is the main port - aka use `psql` to access that.

When you use `pgagroal-cli` to use the management port (`:2347`) -- remote access requires a `pgagroal_admins.conf` file, and an `-A` option upon startup. Otherwise, just use the Unix Domain Socket on a local system (`pgagroal-cli status`).

HTH

**Reply by bokket** [reply]
Hi.To ensure that the variables are consistent, I used pgmonate cluster data, they are basically the same in the tutorial except that pgmonate has an additional repl role to backup to `pgmonate user`, but still `pgagroal user` use passwd to mention `bad credentials for user`, either remote admin access or local system unix socket.But pgmonate succeeded, which confused me until now.
